<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>OperazioniDlg</class>
 <widget class="QDialog" name="OperazioniDlg">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>104</width>
    <height>453</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Gestione Cassa</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <property name="spacing">
    <number>2</number>
   </property>
   <property name="margin">
    <number>10</number>
   </property>
   <item>
    <widget class="QToolButton" name="gestioneBtn">
     <property name="sizePolicy">
      <sizepolicy hsizetype="MinimumExpanding" vsizetype="Expanding">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="text">
      <string>GESTIONE</string>
     </property>
     <property name="icon">
      <iconset resource="risorse.qrc">
       <normaloff>:/GestCassa/gestione</normaloff>:/GestCassa/gestione</iconset>
     </property>
     <property name="iconSize">
      <size>
       <width>48</width>
       <height>48</height>
      </size>
     </property>
     <property name="toolButtonStyle">
      <enum>Qt::ToolButtonTextUnderIcon</enum>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QToolButton" name="cassaBtn">
     <property name="sizePolicy">
      <sizepolicy hsizetype="MinimumExpanding" vsizetype="Expanding">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="text">
      <string>CASSA</string>
     </property>
     <property name="icon">
      <iconset resource="risorse.qrc">
       <normaloff>:/GestCassa/cassa</normaloff>:/GestCassa/cassa</iconset>
     </property>
     <property name="iconSize">
      <size>
       <width>48</width>
       <height>48</height>
      </size>
     </property>
     <property name="toolButtonStyle">
      <enum>Qt::ToolButtonTextUnderIcon</enum>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QToolButton" name="testBtn">
     <property name="sizePolicy">
      <sizepolicy hsizetype="MinimumExpanding" vsizetype="Expanding">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="text">
      <string>TEST</string>
     </property>
     <property name="icon">
      <iconset resource="risorse.qrc">
       <normaloff>:/GestCassa/warning</normaloff>:/GestCassa/warning</iconset>
     </property>
     <property name="iconSize">
      <size>
       <width>48</width>
       <height>48</height>
      </size>
     </property>
     <property name="toolButtonStyle">
      <enum>Qt::ToolButtonTextUnderIcon</enum>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QToolButton" name="stornoBtn">
     <property name="sizePolicy">
      <sizepolicy hsizetype="MinimumExpanding" vsizetype="Expanding">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="text">
      <string>STORNO</string>
     </property>
     <property name="icon">
      <iconset resource="risorse.qrc">
       <normaloff>:/GestCassa/cancella</normaloff>:/GestCassa/cancella</iconset>
     </property>
     <property name="iconSize">
      <size>
       <width>48</width>
       <height>48</height>
      </size>
     </property>
     <property name="autoExclusive">
      <bool>false</bool>
     </property>
     <property name="toolButtonStyle">
      <enum>Qt::ToolButtonTextUnderIcon</enum>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QToolButton" name="statsBtn">
     <property name="sizePolicy">
      <sizepolicy hsizetype="MinimumExpanding" vsizetype="Expanding">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="text">
      <string>STATS</string>
     </property>
     <property name="icon">
      <iconset resource="risorse.qrc">
       <normaloff>:/GestCassa/stats</normaloff>:/GestCassa/stats</iconset>
     </property>
     <property name="iconSize">
      <size>
       <width>48</width>
       <height>48</height>
      </size>
     </property>
     <property name="autoExclusive">
      <bool>false</bool>
     </property>
     <property name="toolButtonStyle">
      <enum>Qt::ToolButtonTextUnderIcon</enum>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QToolButton" name="statsBtn_2">
     <property name="sizePolicy">
      <sizepolicy hsizetype="MinimumExpanding" vsizetype="Expanding">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="text">
      <string>CHIUDE</string>
     </property>
     <property name="icon">
      <iconset resource="risorse.qrc">
       <normaloff>:/GestCassa/freccia_giu_blu</normaloff>:/GestCassa/freccia_giu_blu</iconset>
     </property>
     <property name="iconSize">
      <size>
       <width>48</width>
       <height>48</height>
      </size>
     </property>
     <property name="autoExclusive">
      <bool>false</bool>
     </property>
     <property name="toolButtonStyle">
      <enum>Qt::ToolButtonTextUnderIcon</enum>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="risorse.qrc"/>
 </resources>
 <connections/>
</ui>
